influxdb3/src/commands/create.rs

```diff
@@ -10,25 +12,28 @@ pub struct Config {
 
 #[derive(Debug, clap::Parser)]
 pub enum SubCommand {
-    Token { token: String },
+    Token,
 }
 
 pub fn command(config: Config) -> Result<(), Box<dyn Error>> {
     match config.cmd {
-        SubCommand::Token { token } => {
-            if token.is_empty() {
-                return Err("Token argument must not be empty".into());
-            }
-
+        SubCommand::Token => {
+            let token = {
+                let mut token = String::from("apiv3_");
+                let mut key = [0u8; 64];
+                OsRng.fill_bytes(&mut key);
+                token.push_str(&hex::encode(key));
```

Comment:
Other parts of the platform use base64 rather than hex encoding for the random bits. I recommend doing this both for consistency but also since you'll save 40 characters in the Authorization header with the network traffic:
```
$ echo -n "$(openssl rand -hex 64)"| wc -m
128

$ echo -n "$(openssl rand 64 | base64 -w 0)" | wc -m
88
```